suite-native/device-manager/src/components/DeviceList.tsx

```diff
@@ -140,42 +137,42 @@ export const DeviceList = ({ isVisible, onSelectDevice }: DeviceListProps) => {
         return null;
     }
 
+    const hasNotSelectedDevicesToShow = notSelectedInstancelessDevices.length > 0;
```

Comment:
Please change the name of this prop. It is confusing. I read it as "It has not any device to show" --> there are no such devices.

But it actually should be true if there are some devices. 

What about:
```suggestion
    const hasUnselectedDevices = notSelectedInstancelessDevices.length > 0;
```

or something similar.

Reply:
f312d14b01